pants.toml

```diff
@@ -207,6 +208,9 @@ known_versions = [
   "v2.10.0|linux_x86_64|8ee6ff537341681f9e91bae2d5da451b15c575691e33980893732d866d3cefc4|2301804"
 ]
 
+[pyupgrade]
+args = ["--py38-plus"]
```

Comment:
I think we may be on 3.7 in some places, like CI. We should update those first.

Reply:
Things in CI are set up to honor `.python-version`, so we should be good.

@colin-grapl What specific instances of 3.7 are you seeing and concerned about?